src/pudl/transform/ferc1.py

```diff
@@ -1661,108 +2005,261 @@ def _multiplicative_error_correction(tofix, mask, minval, maxval, mults):
     return fixed
 
 
-##############################################################################
-# DATABASE TABLE SPECIFIC PROCEDURES ##########################################
-##############################################################################
-def plants_steam(ferc1_raw_dfs, ferc1_transformed_dfs):
-    """Transforms FERC Form 1 plant_steam data for loading into PUDL Database.
+def convert_units(df, column_name_new, column_name_old, conversion):
```

Comment:
I wonder if it's possible to automate the unit conversion cleaning function a little bit. 
```python
def convert_units(df, column_name, desired_unit):
    # Infer original unit from column_name
    # convert based on desired_unit
```

Reply:
Yep, I was looking into adding units into the fields metadata because there's a Datasette functionality to handle and display units. I didn't end up adding them but units would be a good future addition.

Reply:
There are a whole lot of dumb units and wild inconsistencies floating around in here, as you can see in the insane gymnastics of the `convert_units()`, `correct_units()` and `standardize_physical_fuel_units()` functions/methods.

I think automating the units conversions probably makes more sense downstream once clean and consistent units are assured.